### PR TITLE
chore: add codeowners and pr template file

### PR DESCRIPTION
**Pull request by lkappeler**
Keep the projects in sync and apply the same codeowners and PR template as for the listings project

### CODE REVIEW

**Comment by ralucacril** [comment]
i dont think the before and after makes sense here.

**Review by ralucacril — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+[Describe the current behavior and / or add a screenshot of the current state]
+
+## After
+
+[Describe the new behavior and / or add a screenshot of the new state]
+
+## How to test
+
+[Add a deep link and instructions how to verify the new behavior]
```
Comment (masone) [inline .github/pull_request_template.md:17]: there's no chance to provide a deep link here

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+[Describe the problem we're solving and lay out the solution]
+
+## Before
+
+[Describe the current behavior and / or add a screenshot of the current state]
+
+## After
+
+[Describe the new behavior and / or add a screenshot of the new state]
```
Comment (masone) [inline .github/pull_request_template.md:13]: before/after makes a lot of sense for visual changes. for such technical project, I'd consider skipping it

**Comment by lkappeler** [comment]
@masone Went with a mininal version of the PR template

**Pull request merged**